src/java/us/kbase/mobu/initializer/test/DockerClientServerTester.java

```diff
@@ -231,369 +231,310 @@ protected static String prepareDockerImage(File moduleDir,
             throw new IllegalStateException("Error building docker image");
         return imageName;
     }
-    
-    protected static void testClients(File moduleDir, String clientEndpointUrl,
-            boolean async, boolean dynamic, String serverType) throws Exception {
-        TypeGeneratorTest.ensureCasperJsInstalled();
```

Comment:
Removed this line from this and the next test and so also removed the JS test code